[2020-11-05 00:04:30,235] INFO [GroupMetadataManager brokerId=0] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2020-11-05 00:14:30,235] INFO [GroupMetadataManager brokerId=0] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2020-11-05 00:15:14,314] WARN Client session timed out, have not heard from server in 6507ms for sessionid 0x17597660f110005 (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:15:14,972] INFO Client session timed out, have not heard from server in 6507ms for sessionid 0x17597660f110005, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:15:16,180] INFO [ZooKeeperClient] Waiting until connected. (kafka.zookeeper.ZooKeeperClient)
[2020-11-05 00:15:16,733] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:15:17,204] INFO Socket connection established to localhost/127.0.0.1:2181, initiating session (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:15:23,207] WARN Client session timed out, have not heard from server in 6004ms for sessionid 0x17597660f110005 (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:15:24,237] INFO Client session timed out, have not heard from server in 6004ms for sessionid 0x17597660f110005, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:15:29,573] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:15:32,585] INFO Socket connection established to localhost/127.0.0.1:2181, initiating session (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:15:38,068] WARN Unable to reconnect to ZooKeeper service, session 0x17597660f110005 has expired (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:15:38,596] INFO Unable to reconnect to ZooKeeper service, session 0x17597660f110005 has expired, closing socket connection (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:15:38,681] INFO EventThread shut down for session: 0x17597660f110005 (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:15:38,681] INFO [ZooKeeperClient] Session expired. (kafka.zookeeper.ZooKeeperClient)
[2020-11-05 00:15:40,543] WARN Attempting to send response via channel for which there is no open connection, connection id 127.0.1.1:9092-127.0.0.1:59180-4 (kafka.network.Processor)
[2020-11-05 00:15:41,048] INFO [ZooKeeperClient] Initializing a new session to localhost:2181. (kafka.zookeeper.ZooKeeperClient)
[2020-11-05 00:15:41,658] INFO Initiating client connection, connectString=localhost:2181 sessionTimeout=6000 watcher=kafka.zookeeper.ZooKeeperClient$ZooKeeperClientWatcher$@130d63be (org.apache.zookeeper.ZooKeeper)
[2020-11-05 00:15:42,344] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:15:42,654] INFO Socket connection established to localhost/127.0.0.1:2181, initiating session (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:15:44,267] INFO Session establishment complete on server localhost/127.0.0.1:2181, sessionid = 0x17597660f110014, negotiated timeout = 6000 (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:15:47,872] INFO Creating /brokers/ids/0 (is it secure? false) (kafka.zk.KafkaZkClient)
[2020-11-05 00:15:48,854] WARN Client session timed out, have not heard from server in 4000ms for sessionid 0x17597660f110014 (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:15:49,117] INFO Client session timed out, have not heard from server in 4000ms for sessionid 0x17597660f110014, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:15:49,516] INFO [ZooKeeperClient] Waiting until connected. (kafka.zookeeper.ZooKeeperClient)
[2020-11-05 00:15:49,931] INFO [ZooKeeperClient] Connected. (kafka.zookeeper.ZooKeeperClient)
[2020-11-05 00:15:51,071] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:15:51,510] INFO Socket connection established to localhost/127.0.0.1:2181, initiating session (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:15:52,529] INFO Session establishment complete on server localhost/127.0.0.1:2181, sessionid = 0x17597660f110014, negotiated timeout = 6000 (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:15:53,326] INFO Result of znode creation at /brokers/ids/0 is: OK (kafka.zk.KafkaZkClient)
[2020-11-05 00:15:53,536] INFO Registered broker 0 at path /brokers/ids/0 with addresses: ArrayBuffer(EndPoint(pranjay-HP-Laptop-15q-bu0xx,9092,ListenerName(PLAINTEXT),PLAINTEXT)) (kafka.zk.KafkaZkClient)
[2020-11-05 00:15:54,081] INFO Creating /controller (is it secure? false) (kafka.zk.KafkaZkClient)
[2020-11-05 00:15:54,511] INFO Result of znode creation at /controller is: OK (kafka.zk.KafkaZkClient)
[2020-11-05 00:16:21,427] INFO [ReplicaAlterLogDirsManager on broker 0] Added fetcher for partitions List() (kafka.server.ReplicaAlterLogDirsManager)
[2020-11-05 00:19:13,442] WARN Client session timed out, have not heard from server in 4777ms for sessionid 0x17597660f110014 (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:19:15,061] INFO Client session timed out, have not heard from server in 4777ms for sessionid 0x17597660f110014, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:19:18,057] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:19:19,209] INFO Socket connection established to localhost/127.0.0.1:2181, initiating session (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:19:22,087] WARN Unable to reconnect to ZooKeeper service, session 0x17597660f110014 has expired (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:19:22,997] INFO Unable to reconnect to ZooKeeper service, session 0x17597660f110014 has expired, closing socket connection (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:19:22,094] INFO [ZooKeeperClient] Session expired. (kafka.zookeeper.ZooKeeperClient)
[2020-11-05 00:19:22,094] INFO EventThread shut down for session: 0x17597660f110014 (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:19:24,314] INFO [ZooKeeperClient] Initializing a new session to localhost:2181. (kafka.zookeeper.ZooKeeperClient)
[2020-11-05 00:19:25,532] INFO Initiating client connection, connectString=localhost:2181 sessionTimeout=6000 watcher=kafka.zookeeper.ZooKeeperClient$ZooKeeperClientWatcher$@130d63be (org.apache.zookeeper.ZooKeeper)
[2020-11-05 00:19:25,810] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:19:26,000] INFO Socket connection established to localhost/127.0.0.1:2181, initiating session (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:19:26,565] INFO Session establishment complete on server localhost/127.0.0.1:2181, sessionid = 0x17597660f110015, negotiated timeout = 6000 (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:19:26,956] INFO Creating /brokers/ids/0 (is it secure? false) (kafka.zk.KafkaZkClient)
[2020-11-05 00:19:27,331] INFO Result of znode creation at /brokers/ids/0 is: OK (kafka.zk.KafkaZkClient)
[2020-11-05 00:19:27,596] INFO Registered broker 0 at path /brokers/ids/0 with addresses: ArrayBuffer(EndPoint(pranjay-HP-Laptop-15q-bu0xx,9092,ListenerName(PLAINTEXT),PLAINTEXT)) (kafka.zk.KafkaZkClient)
[2020-11-05 00:19:27,799] INFO Creating /controller (is it secure? false) (kafka.zk.KafkaZkClient)
[2020-11-05 00:19:28,580] INFO Result of znode creation at /controller is: OK (kafka.zk.KafkaZkClient)
[2020-11-05 00:19:58,462] INFO [ReplicaAlterLogDirsManager on broker 0] Added fetcher for partitions List() (kafka.server.ReplicaAlterLogDirsManager)
[2020-11-05 00:19:59,357] INFO Terminating process due to signal SIGHUP (kafka.Kafka$)
[2020-11-05 00:19:59,358] INFO Terminating process due to signal SIGHUP (kafka.Kafka$)
[2020-11-05 00:20:00,051] INFO [KafkaServer id=0] shutting down (kafka.server.KafkaServer)
[2020-11-05 00:20:00,939] INFO [KafkaServer id=0] Starting controlled shutdown (kafka.server.KafkaServer)
[2020-11-05 00:20:05,648] WARN Session 0x17597660f110015 for server localhost/127.0.0.1:2181, unexpected error, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:379)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:68)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:366)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)
[2020-11-05 00:20:06,585] INFO [ZooKeeperClient] Waiting until connected. (kafka.zookeeper.ZooKeeperClient)
[2020-11-05 00:20:07,205] INFO [ZooKeeperClient] Connected. (kafka.zookeeper.ZooKeeperClient)
[2020-11-05 00:20:08,426] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:20:09,055] WARN Session 0x17597660f110015 for server null, unexpected error, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
java.net.ConnectException: Connection refused
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:716)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)
[2020-11-05 00:20:09,512] INFO [ZooKeeperClient] Waiting until connected. (kafka.zookeeper.ZooKeeperClient)
[2020-11-05 00:20:10,526] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:20:11,610] WARN Session 0x17597660f110015 for server null, unexpected error, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
java.net.ConnectException: Connection refused
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:716)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)
[2020-11-05 00:20:13,242] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:20:13,464] WARN Session 0x17597660f110015 for server null, unexpected error, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
java.net.ConnectException: Connection refused
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:716)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)
[2020-11-05 00:20:15,673] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:20:26,175] WARN Session 0x17597660f110015 for server null, unexpected error, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
java.net.ConnectException: Connection refused
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:716)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)
[2020-11-05 00:20:35,316] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-11-05 00:20:38,448] WARN Session 0x17597660f110015 for server null, unexpected error, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
java.net.ConnectException: Connection refused
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:716)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)
[2020-11-05 00:20:41,284] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
